learners/setup.md

```diff
@@ -2,53 +2,138 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+::::::::::::::::::::::::::::::prereq
 
-## Data Sets
+In this lesson we use Git from the Unix Shell. Some previous experience with the shell is expected, but isn’t mandatory.
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+::::::::::::::::::::::::::::::
+
+## Computer
+
+Participants must work on a computer with a Mac, Linux, or Windows operating system (not a tablet, Chromebook, etc.) on which they have administrative privileges.
 
 ## Software Setup
 
-::::::::::::::::::::::::::::::::::::::: discussion
+To participate in this workshop, you will need to prepare the following (if you haven't already):
 
-### Details
+#### Install Shell and Git. 
+   
+:::tab
 
-Setup for different systems can be presented in dropdown menus via a `spoiler`
-tag. They will join to this discussion block, so you can give a general overview
-of the software used in this lesson here and fill out the individual operating
-systems (and potentially add more, e.g. online setup) in the solutions blocks.
+### Windows
+Git should be installed on your computer as part of your Bash install.
 
-:::::::::::::::::::::::::::::::::::::::::::::::::::
 
-:::::::::::::::: spoiler
+### Mac  
+Mac
```

Comment:
Most versions of MacOS  will have `git` preinstalled, you can check by running `git version` in the terminal. If it is not available, you can install `git` through [homebrew](https://brew.sh/) by running:
`brew install git`